airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -357,11 +357,11 @@ def request_params(self, stream_state=None, **kwargs):
         if stream_state.get(self.cursor_field):
             issue_worklogs_state = pendulum.parse(stream_state.get(self.cursor_field))
             state_row = int(issue_worklogs_state.timestamp() * 1000)
-            params["startedAfter"] = state_row
+            params[self.cursor_field] = state_row
```

Comment:
this is wrong, cursor_field is the field in record and state object

Reply:
also did you check that this works? 
even if there is such parameter, we need to get all records with updated >= state_row, not just updated == state_row. So please make sure it works

Reply:
@po3na4skld 
I don't see you reacted to this comment. The problem with this code is that we can't use the name of cursor_field as the name for request parameters, because they are completely unrelated, even if their values are the same